appservice/src/intTest/groovy/nu/yona/server/EditGoalsTest.groovy

```diff
@@ -45,10 +45,6 @@ class EditGoalsTest extends AbstractAppServiceIntegrationTest
 		response.status == 200
 		response.responseData._links?.self.href == richard.url + appService.GOALS_PATH_FRAGMENT
 		response.responseData._embedded.budgetGoals.size() == 2
-		response.responseData._embedded.budgetGoals[1].activityCategoryName == 'gambling'
```

Comment:
I applied this change directly on the master, to get the tests green again. See this change set: https://github.com/yonadev/yona-server/commit/6215e34b37c48dd25e3f1cd00ce3600a3a9b886e